webpack/assets/javascripts/react_app/components/DiffView/DiffRadioButtons.js

```diff
@@ -21,8 +21,12 @@ const radioButton = (stateView, btnView, changeState) => (
 const DiffRadioButtons = ({ stateView, changeState }) => (
   <div id="diff-radio-buttons">
     <ButtonGroup>
-      {radioButton(stateView, __('split'), changeState)}
-      {radioButton(stateView, __('unified'), changeState)}
+      {radioButton(stateView, __('split'), () => {
+        changeState('split');
```

Comment:
can you change the string to a constant?  
import { SPLIT, UNIFIED } from './DiffConsts.js
```js
radioButton(stateView, __('split'), () => {
      {radioButton(stateView, __('unified'), () => changeState(SPLIT);
}
```js

Reply:
Changed, thanks!